clang/lib/Sema/SemaType.cpp

```diff
@@ -7438,6 +7438,21 @@ static bool HandleWebAssemblyFuncrefAttr(TypeProcessingState &State,
   return false;
 }
 
+static void HandleSwiftAttr(TypeProcessingState &State, QualType &QT,
+                            ParsedAttr &PAttr) {
+  Sema &S = State.getSema();
+
+  StringRef Str;
+  if (!S.checkStringLiteralArgumentAttr(PAttr, 0, Str)) {
+    PAttr.setInvalid();
+    return;
+  }
+
```

Comment:
I did some more work on this recently and this is not that simple, I need to figure out how to distinguish between when the attribute should be handled as a declaration attribute vs. type attribute. I have a few ideas already.